app/components/projects/galleries/complementary.js

```diff
@@ -22,7 +22,7 @@ const COMPLEMENTARY = [
     href: "#complementar",
     title: "Consultoria Complementar",
     category1: "concept board",
-    category2: "Sala",
+    category2: "  Cozinha",
```

Comment:
is the extra space needed?

Reply:
no, well noted! Thank you.
Extra space was deleted.